rebar.config

```diff
@@ -6,8 +6,7 @@
 
 { deps, [
     {amoc_arsenal, {git, "https://github.com/esl/amoc-arsenal.git", {branch, "main"}}},
-    {escalus, "4.2.16"},
-    {exml, "3.4.1", {pkg, hexml}},
```

Comment:
We use this exml library directly, and it is a bad practice to depend on a "dependency of a dependency". We should be declaring our dependencies here strictly. You can do something like
```erlang
    {exml, "~> 4.0", {pkg, hexml}},
```
To ensure we declare "any" version higher than 4.0, and then let escalus choose something more precise if needed.

Reply:
Thanks for the comment, I've fixed it ;)